include/clink/linalg/sparse_vector.h

```diff
@@ -42,11 +40,14 @@ class SparseVector {
   llvm::Error set(const int index, const double value);
 
   // Gets the value at a certain index of the vector.
-  llvm::Expected<double> get(const int index);
+  llvm::Expected<double> get(const int index) const;
 
   // Gets the total number of dimensions of the vector.
   int size();
 
+  // Gets a string representation of this vector.
+  std::string toString();
```

Comment:
Is this method only needed by test? If so, could we put it in test_utils?

Note that `SparseVector` is conceptually similar to `CooHostTensor` in TFRT. `CooHostTensor` has `void Print(raw_ostream& os) const override` instead of `toString`. `Print(...)` is more efficient than `toString` when the vector is large. Could we follow TFRT's practice?

Reply:
TFRT uses MLIR's tools for testing, which can directly check result string from the output raw stream. Given that we're testing in C++, I now think we need not convert the `SparseVector` to string and print it out or return from MLIR script. I'll add a `bool operator==(...)` method to `SparseVector` so we can directly do `EXPECT_EQ(expected_vector, actual_vector`.

Reply:
Sounds good.